portality/view/doaj.py

```diff
@@ -423,6 +423,11 @@ def terms():
     return render_template("layouts/static_page.html", page_frag="/legal/terms.html")
 
 
+@blueprint.route("/media/")
+def media():
+    return render_template("layouts/static_page.html", page_frag="/legal/media.html")
```

Comment:
We haven't feature annotated the routes yet, but since we're changing this one, we should add the docstring

```
"""
~~Media:WebRoute~~
"""
```

Reply:
Like this?

````
@blueprint.route("/media/")
def media():
    """
    ~~Media:WebRoute~~
    """
    return render_template("layouts/static_page.html", page_frag="/legal/media.html")
````

...my Atom keeps giving me errors when I do that